ui/app/components/clients/history.js

```diff
@@ -179,7 +180,7 @@ export default class History extends Component {
       }
       this.queriedActivityResponse = response;
     } catch (e) {
-      // ARG TODO handle error
+      return e;
```

Comment:
`return e` will return an error-shaped object which you have to check for, while `throw e` will allow you to wrap the consumer in a try/catch. If you do want to throw e you actually don't need to wrap this method in a try/catch because it will throw automatically. Just some notes when you think through how you want to handle errors